eth/tracers/native/call.go

```diff
@@ -166,6 +166,11 @@ func (t *callTracer) CaptureState(pc uint64, op vm.OpCode, gas, cost uint64, sco
 		stack := scope.Stack
 		stackData := stack.Data()
 
+		stackRequired := (vm.LOG4 - vm.LOG0) + 2
```

Comment:
I think `stackRequired` should depend on the specific LOG instruction

Reply:
Yes.  good catch.